pom.xml

```diff
@@ -2,30 +2,80 @@
 	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
 	<modelVersion>4.0.0</modelVersion>
 	<groupId>ch.cern.eos</groupId>
-	<version>0.1.0</version>
-	<artifactId>hadoop-xrootd</artifactId>
-
+	<version>0.0.1-SNAPSHOT</version>
+	<artifactId>EOSConnector</artifactId>
+    <packaging>nar</packaging>
 	<repositories>
 		<repository>
 			<id>cloudera-repo-releases</id>
 			<url>https://repository.cloudera.com/artifactory/repo/</url>
 		</repository>
-	</repositories>
+    </repositories>
+    <properties>
+        <maven.compiler.source>1.7</maven.compiler.source>
+        <maven.compiler.target>1.7</maven.compiler.target>
+    </properties>
 	<dependencies>
 		<dependency>
 			<groupId>org.apache.hadoop</groupId>
 			<artifactId>hadoop-common</artifactId>
-			<version>2.7.4</version>
```

Comment:
This we would need to test. We are likely to look into 2.7.6

Reply:
I think if/when it's moved to being built by CI, there will probably need to be builds against each major hadoop release and probably one "hadoop-provided" release that doesn't have any dependency